phovea.js

```diff
@@ -5,70 +5,100 @@
  **************************************************************************** */
 
 //register all extensions in the registry following the given pattern
-module.exports = function(registry) {
+module.exports = function (registry) {
```

Comment:
sorry, I saw the same format in another phovea.js file and thought this one would be more readable. I won't reformat all the code in the future.